src/librustdoc/html/layout.rs

```diff
@@ -70,6 +70,10 @@ r##"<!DOCTYPE html>
         {sidebar}
     </nav>
 
+    <div id="theme-picker">&#x1f58c;
```

Comment:
This icon is not present in the fonts we bundle with rustdoc, nor in the system fonts in Windows 7:

![image](https://user-images.githubusercontent.com/5217170/35231864-85bbf648-ff5f-11e7-85dc-e4ae9fda0f2f.png)